### PR TITLE
Add sigh in page

### PR DESCRIPTION
**Pull request by AnyaLyaLya**
https://fe-feb23-code-hunters.github.io/fe_phone-catalog/#/auth

### CODE REVIEW

**Review by Hamuud — APPROVED** [review]
Looks great, i check on phone, and all form looks amazing

File: src/components/SignInSide/SignInSide.tsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+import LockOutlined from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import './signInSide.scss';
+import { teal } from '@mui/material/colors';
+import { useState } from 'react';
+
+function Copyright(props: any) {
+  return (
+    <Typography
```
Comment (prokivskaaa) [inline src/components/SignInSide/SignInSide.tsx:20]: Maybe let's make it reusable?

File: src/components/SignInSide/SignInSide.tsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+});
+
+const SignInSide = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const [emailError, setEmailError] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+
+  const validatePassword = (pw: string) => {
```
Comment (prokivskaaa) [inline src/components/SignInSide/SignInSide.tsx:71]: let's make validate func global in utils and use it where needed

File: src/components/SignInSide/SignInSide.tsx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    const pwRegex = /^(?=.*[A-Z])(?=.*\d)(?!.*[^\w\s])(?=.*[a-z]).{8,16}$/;
+
+    return pwRegex.test(pw);
+  };
+
+  const validateEmail = (em: string) => {
```
Comment (prokivskaaa) [inline src/components/SignInSide/SignInSide.tsx:77]: The same

File: src/components/SignInSide/SignInSide.tsx [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const isEmailValid = validateEmail(email);
+    const isPasswordValid = validatePassword(password);
+
+    if (isEmailValid && isPasswordValid) {
+      // Виконайте потрібні дії для входу користувача
```
Comment (prokivskaaa) [inline src/components/SignInSide/SignInSide.tsx:90]: Delete comment

**Review by prokivskaaa — CHANGES_REQUESTED** [review]
A few comments:

- After sign up redirect to the main page
- Let's leave on sign up only fields for email and password as we don't show it anywhere and we do not have BE functionality for it
- Now I see the problem with the transition. Let's take away bold text oh hover from everywhere (Sorry, did not get the point at first)
- Let's make the same indents between the password and Sign in button 
telegram-cloud-photo-size-2-5228934129976985495-y
- Let's make texts "Forgon password" and "Don't have an account" smaller for all the devices and let's fix the distance between the texts on adaptation (make it bigger) 
telegram-cloud-photo-size-2-5228934129976985490-y
telegram-cloud-photo-size-2-5228934129976985489-y
- Let's make the text "I want to receive inspiration, marketing promotions and updates via email." and "Already have an account? Sign in" smaller everywhere
Знімок екрана 2023-06-20 о 16 41 55

**Pull request merged**